### PR TITLE
multi: bump all dependency to use lnd 0.15.0-beta.rc4

### PR DESCRIPTION
**Pull request by guggero**
This commit bumps all dependencies to be in line with what's required
for the current release candidate of lnd 0.15.0-beta.

This enables LiTd to be bundled with the latest version of lnd and loop.

Some commits were taken from https://github.com/lightninglabs/loop/pull/477.

#### Pull Request Checklist
- [ ] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

**Comment by guggero** [comment]
cc @dstadulis (part of LiT update I mentioned).

**Review by bhandras — APPROVED** [review]
LGTM :+1:

File: go.mod [hunk 1]
```diff
@@ -13,17 +13,17 @@ require (
...
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/aperture v0.1.17-beta.0.20220328072456-4a2632d0be38
-	github.com/lightninglabs/lndclient v0.15.0-0
-	github.com/lightninglabs/loop/swapserverrpc v1.0.0
+	github.com/lightninglabs/aperture v0.1.18-beta
+	github.com/lightninglabs/lndclient v0.15.0-6
+	github.com/lightninglabs/loop/swapserverrpc v1.0.1
 	github.com/lightninglabs/protobuf-hex-display v1.4.3-hex-display
-	github.com/lightningnetwork/lnd v0.14.1-beta.0.20220328072343-6e2214fd0fc0
+	github.com/lightningnetwork/lnd v0.15.0-beta.rc3
```
Comment (bhandras) [inline go.mod:20]: We might as well bump to rc4?

**Review by sputn1ck — APPROVED** [review]
LGTM 🚀

**Pull request merged**